CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Unify field names on product, collection and page - #3706 by @michaljelonek
 - Generate voucher code if it wasn't provided in mutation - #3717 by @Kwaidan00
 - Reuse Storefront's 1.0 payment logic in API - #3715 by @maarcingebala
+- Fix logo size in different browser and devices with different sizes -3722 # by @jxltom
```

Comment:
I think we are tagging issues differently

Reply:
> I think we are tagging issues differently

Thanks! Fixed!